### DISCUSSION TITLE
I cant sign my project

### DISCUSSION THREAD

**Discussion by Omidava** [Sketchware-Pro/Sketchware-Pro#704] [Q&A] [answered]
When I sign the project I have error about release key

**Comment by j-px** [comment]
Can you at least share that error message here? We can't read minds or just guess whatever it might be

**Comment by Omidava** [comment]
![Screenshot_2023-01-11-10-23-19~2](https://user-images.githubusercontent.com/121393174/211738498-b9db8bf3-a6b5-44ed-b474-7fc2a333dd1f.png)

see this image

**Comment by j-px** [comment, marked as answer]
Please read the dialog's message:  
![Screenshot_20230111_232022_Sketchware Pro](https://user-images.githubusercontent.com/40118124/211930397-db3365c2-8c43-4973-b7dc-54d773d3a1c3.png)

Also maybe consider exporting an unsigned APK and signing that file with another tool, for example apk-signer or MT Manager instead of signing with Sketchware Pro.
